README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## **About Me**
+
+Hello, My name is **Tianhong Yin**, an MA student in international relations at the University of Chicago.
+
+My **research interests** include:
+* Conflict Studies
+* American Foreign Policy
+* American Politics
+* Quantitative Methodology
+
+I am a cat person and I love to know everything about cats during my spare time. Here is the [link to a cat forum](http://thecatsite.com) I often visit to learn about cats.
```

Comment:
I like the way you added a link and renamed it for ease of use

Reply:
One thing that's more typical style is to omit the "link to" part of the link text. You could just say "Here is a [cat forum] I go to . . .  
That's just a minor style option, though. I couldn't think of anything else. Looks great!

Reply:
I would echo this point, but ultimately as your peer said, it is a minor styling choice.

Reply:
Your suggestions are useful, and I will pay attention to that in future work. Thanks!